app/http.php

```diff
@@ -65,22 +66,36 @@
 
 include __DIR__ . '/controllers/general.php';
 
-$domain = App::getEnv('_APP_DOMAIN', '');
-
-Console::info('Issuing a TLS certificate for the master domain ('.$domain.') in 30 seconds.
-    Make sure your domain points to your server IP or restart your Appwrite server to try again.'); // TODO move this to installation script
+function certificateSetup($domain)
+{
+    if($domain=='localhost' || (bool)ip2long($domain)) return;
+    $domains = Config::getParam('domains', []);
+    if (!in_array($domain, $domains)) {
```

Comment:
array_key_exists check will be faster than in_array. You can have an array as follow:
[
    'example.com' => true,
    'example2.com' => true,
    'example3.com' => true,
]